src/actions/profile-view.js

```diff
@@ -4,7 +4,6 @@
 
 // @flow
 import { oneLine } from 'common-tags';
-import { getLastVisibleThreadTabSlug } from '../selectors/app';
```

Comment:
Here's a good example of complicated split up imports that are simplified with a single import.